### PR TITLE
Run tests in wheel

### PR DESCRIPTION
**Pull request by gigiblender**
This PR runs the python unit tests as part of the nightly build. 

Since at the moment there are some failing tests, I do an `|| true` to ignore the return code of the `run_tests.sh` command and [keep the green tick-mark](https://github.com/actions/toolkit/issues/399) in the actions section.

@areusch

### CODE REVIEW

File: docker/Dockerfile.package-cpu [hunk 1]
```diff
@@ -27,17 +27,16 @@ RUN bash /install/centos_install_arm_ethosn_driver_stack.sh
 COPY install/centos_install_arm_compute_library.sh /install/centos_install_arm_compute_library.sh
 RUN bash /install/centos_install_arm_compute_library.sh
 
+# Install Conda
```
Comment (areusch) [inline docker/Dockerfile.package-cpu:30]: just curious, why this change?

**Review by areusch — COMMENTED** [review]
thanks @gigiblender !

File: docker/Dockerfile.package-cpu (hunk 1 above)
Comment (gigiblender) [reply docker/Dockerfile.package-cpu:30]: [centos_install_python_packages](https://github.com/tlc-pack/tlcpack/blob/9928a5d01ca4b692645953917af25a95ab13b92c/docker/install/centos_install_python_package.sh#L18) will install the required python packages to run the tests for the current Conda environment. We need to first install Conda before doing that.

**Review by areusch — APPROVED** [review]

**Comment by areusch** [comment]
sorry; approved the GH workflows to run, let's merge depending on how they look

**Comment by gigiblender** [comment]
> sorry; approved the GH workflows to run, let's merge depending on how they look

Seems like I need to build new docker images.

**Comment by areusch** [comment]
@gigiblender also i think the workflows are silently faling. i think we discuss this was in case we need to patch up some tests...if they are all passing maybe we should remove `|| true` and just start failing workflows when tests fail?

**Comment by driazati** [comment]
for the failing tests maybe we should file issues and add a `pytest.mark.skipIf(os.getenv("WHEEL_TEST", "")...)` on the actual test in apache/tvm so we can avoid the silent failures

**Comment by gigiblender** [comment]
@areusch @driazati. As advised, I created https://github.com/apache/tvm/pull/11705 and removed the `|| true` condition. 

Could you please have another look? 
I will create an issue listing the failing tests as well.

**Review by areusch — APPROVED** [review]

**Comment by areusch** [comment]
thanks @gigiblender , @driazati want to have a look?

File: .github/workflows/wheel_manylinux_nightly.yaml [hunk 2]
```diff
@@ -42,6 +42,13 @@ jobs:
         CUDA: ${{ matrix.config.cuda }}
       run: |
         docker/bash.sh --no-gpu $IMAGE ./wheel/build_wheel_manylinux.sh --cuda $CUDA
+    - name: Test
+      env:
+        IMAGE: ${{ matrix.config.image }}
+        WHEEL_TEST: "WHEEL_TEST"
+      continue-on-error: true
```
Comment (driazati) [inline .github/workflows/wheel_manylinux_nightly.yaml:49]:
```suggestion
```
do we still need this now that the skips are in place in apache/tvm?

**Review by driazati — COMMENTED** [review]
seems like there are hidden failures in the CI run:

https://github.com/tlc-pack/tlcpack/runs/6885670232?check_suite_focus=true#step:6:112

once that's fixed and the tests run this is probably good to go

File: .github/workflows/wheel_manylinux_nightly.yaml (hunk 2 above)
Comment (driazati) [reply .github/workflows/wheel_manylinux_nightly.yaml:49]: ah I see, yeah keeping it makes sense then. It'd be nicer to assert that no new tests broke but that seems like it'd be more complicated than it's worth here. The tests still error out complaining about `junitparser` missing before running though so that needs to be fixed before we can merge this

**Comment by gigiblender** [comment]
Ok, I think this is all good now.
